src/graph/Edge.js

```diff
@@ -104,6 +105,7 @@
 
     Edge.prototype.update = function (domNode, element, transitionDuration, skipPushMarkers) {
         SVGWidget.prototype.update.apply(this, arguments);
+        element.classed(this.classed());
```

Comment:
whats a proper use example of this publish parameter? you have an object specified but i d3 selection classed doesnt take an obj as far as i know and i cant find the this.classed() function either